packages/app/features/player/fullScreenPlayer.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, Text, View } from "app/design-system";
+import { Pressable } from "app/design-system";
+import { tw } from "app/design-system/tailwind";
+import ChevronDown from "app/ui/icons/chevron-down";
+import { Dimensions } from "react-native";
+import { LikesList } from "./components/likesLIst";
+import { PlayerButtonsRow } from "./components/playerButtonsRow";
+import { PlayerSlider } from "./components/playerSlider";
+import { SafeAreaView } from "app/design-system/safe-area-view";
+
+const { height } = Dimensions.get("window");
+
+type Props = {
+  onTogglePress: () => void;
+};
+
+export function FullScreenPlayer({ onTogglePress }: Props) {
+  return (
+    <SafeAreaView
+      className={`flex flex-column px-4 bg-blue-dark items-center absolute w-full h-${
```

Comment:
```suggestion
      className={`flex flex-col px-4 bg-blue-dark items-center absolute w-full h-${
```

You can drop flex-col, flex direction defaults to column